### DISCUSSION TITLE
Website

### DISCUSSION THREAD

**Discussion by a69y** [JoseskVolpe/UndertaleME#8] [General]
I Am Learning Web Development!, I Visited Webpage Of Your Project And It Looks Ugly Can I Make A Good Web For Your Project?

**Comment by JoseskVolpe** [comment]
Ohh hi ^w^
Sorry for the huge delay
Sure, you can :3
¡Your contribution will be really appreciated!

**Comment by JoseskVolpe** [comment]
Right now i'm rewritting huge part of the code (almost the entire project from scratch), i've planned to make a core engine for it recently and to integrate it to Hecl language, i also planned to make it to read zipped UndertaleME projects .... So in terms, the little ugly website i made is outdated aswell lol

I would need an download page, source-code redirector, projects page and Wiki documentation page. You may also include some more stuff if you wish, and of course, the bad Sans's style jokes too x3

**Comment by a69y** [comment]
Okkk! :), Btw, Should I Made Using Bootstrap or By Plain CSS?, And Also It Should Be Viewable On Mobile Devices (Java) or Not?

**Reply by JoseskVolpe** [reply]
GitHub uses modern HTTPS encryption wich makes it impossible to be accessed on most default Java devices browser, however Opera Mini and UCBrowser are able to access them by proxy. So yeah, it'll be better if it'll be viewable on Java devices, so bootstrap may not be a good option on this case since it relies on real-time rendering and may cause incompatibility on these devices, but some of bootstrap's features can still be used as these phones would have no or minor problems with them.
We will need to test to be sure that the website is still accessible and viewable.

**Comment by a69y** [comment]
Hi, Why Are You Not Online At Dedomil, People Miss You A Lot! :(
Btw, Website Development Is In Progress!!.

**Reply by a69y** [reply]
No Reply? @JoseskVolpe
